src/coreclr/tools/aot/nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- For LLVMSharp -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This will fail in official builds. I will need to ask for the required packaged mirrored in the dotnet-public feed, and then this nuget.config can be deleted.

Reply:
ok, I will convert to draft.  When/if the package is mirrored I will remove the file and try another CI run.  Thanks

Reply:
The following packages have been added to the public feed:

LLVMSharp 15.0.0-beta1
libLLVM 15.0.0 
libLLVM.runtime.win-x64 15.0.0
libLLVM.runtime.ubuntu.20.04-x64 15.0.0
libLLVM.runtime.osx-x64 15.0.0
libLLVM.runtime.ubuntu.18.04-x64 15.0.0
libLLVM.runtime.win-x86 15.0.0
libLLVM.runtime.freebsd.11-x86 12.0.0
libLLVM.runtime.freebsd.11-x64 12.0.0
libLLVM.runtime.linux-arm64 15.0.0 
libLLVM.runtime.win-arm64 15.0.0
libLLVM.runtime.freebsd.13-x64 15.0.0
libLLVM.runtime.freebsd.12-x64 15.0.0
libLLVM.runtime.linux-x64 15.0.0 
libLLVM.runtime.osx-arm64 15.0.0 
libLLVM.runtime.ubuntu.22.04-x64 15.0.0

You should be able to delete this nuget.config now.

Reply:
Please could you request LLVMSharp.Interop 15.0.0-beta1 as well?

Reply:
LLVMSharp.Interop should be there now